packages/javascript/bh-shared-ui/src/views/TierManagement/Details/MembersList.test.tsx

```
@@ -31,31 +31,35 @@ beforeAll(() => server.listen());
     it('sorting the list updates the list by changing the call made to the API', async () => {
         const user = userEvent.setup();
 
         const history = createMemoryHistory({
-            initialEntries: ['/tier-management/details/tag/1/selector/1'],
+            initialEntries: ['/tier-management/details/tier/1/selector/1'],
         });
 
         render(
             <Routes>
                 <Route path={'/'} element={<MembersList selected='1' onClick={vi.fn()} />} />;
                 <Route
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)